subworkflows/local/qc_bam.nf

```diff
@@ -14,6 +14,7 @@ workflow QC_BAM {
     take:
         bam              // channel: [ val(meta), path(bam) ]
         bai              // channel: [ val(meta), path(bai) ]
+        bam_bai
```

Comment:
We could update the mosdepth part to take the mixed bam bai channel as well

Reply:
True I will create a note and fix it in another PR